app/models/pro_account.rb

```diff
@@ -14,6 +14,8 @@
 class ProAccount < ApplicationRecord
   include AlaveteliFeatures::Helpers
 
+  attr_writer :source
```

Comment:
I don't really understand why we need to do this, and the commit message doesn't explain.

Reply:
You're right this doesn't make sense in this PR - its for the next one so will bump this commit there

Reply:
Removed in 1df9737e21f788e3fb569e878753cd2a54400374. Will remove the commits before merge.